### PR TITLE
added excel support and fixed conn_string

### PR DESCRIPTION
**Pull request by wagnerlmichael**
I made an if statement to add support for excel files, and I also changed the conn_string variable from 

f"postgres://{user}:{passw}@{host}/{db_name}" to 

f"postgresql://{user}:{passw}@{host}/{db_name}"

as the former was throwing an error.

### CODE REVIEW

File: upload_df/df_tosql.py [hunk 1]
```diff
@@ -4,33 +4,35 @@
...
+    Supported formats: csv, xlsx
     Input: Psql user, password, host name, database name, table name,
     upload option (strings)
     Output: None
     """
 
-    df=pd.read_csv(file_path)
+    if file_path.endswith(".csv"):
+        df = pd.read_csv(file_path)
+    elif file_path.endswith(".xlsx"):
```
Comment (mgutierrezc) [inline upload_df/df_tosql.py]: Could we also add support for .xls files?

**Review by mgutierrezc — CHANGES_REQUESTED** [review]
Great! Just added a small request

File: upload_df/df_tosql.py (hunk 1 above)
Comment (wagnerlmichael) [reply upload_df/df_tosql.py]: For sure, good point. Just pushed support for .xls to the branch.

File: upload_df/df_tosql.py [hunk 2]
```diff
@@ -4,33 +4,35 @@
 import psycopg2
 
 
-
 def upload_to_psql(file_path, user, passw, host, db_name, table_name, option):
     """
-    Uploads csv data to psql
-
+    Uploads data file to psql
+    Supported formats: csv, xlsx
```
Comment (mgutierrezc) [inline upload_df/df_tosql.py:10]: And xls

**Review by mgutierrezc — APPROVED** [review]

**Pull request merged**